### PR TITLE
ci: set up covector

### PR DESCRIPTION
**Pull request by FabianLars**
I saw #53 too late but this PR has a few fixes more (plus more local testing probably) anyway so whatever 🤷

### CODE REVIEW

File: .changes/config.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          "dryRunCommand": true,
+          "pipe": true
+        }
+      ]
+    }
+  },
+  "packages": {
+    "authenticator": {
+      "path": "./plugins/authenticator",
+      "manager": ""
```
Comment (amrbashir) [inline .changes/config.json:38]: why aren't you specifying a manager?

Comment (FabianLars) [reply .changes/config.json:38]: so that covector skips it since we don't want to publish it yet

Comment (FabianLars) [reply .changes/config.json:38]: would be prettier if json would support comments :D (or if covector would have an explicit `disabled` setting maybe? idk)

Comment (amrbashir) [reply .changes/config.json:38]: ah, well, everything looks fine to me

**Comment by FabianLars** [comment]
Thanks Amr :) Since this whole repo is like one big experiment i'll just go ahead and see what happens 😂

**Pull request merged**